se/commands/find_unusual_characters.py

```diff
@@ -0,0 +1,133 @@
+	return_code = 0
+	unusual_characters: Dict[str, int] = {} # key: word; value: count
+	target_filenames = se.get_target_filenames(args.targets, ".xhtml")
+	files_xhtml = []
+
+	# Read files and cache for later
+	for filename in target_filenames:
+		try:
+			with open(filename, "r", encoding="utf-8") as file:
+				xhtml = file.read()
+				dom = se.easy_xml.EasyXmlTree(xhtml)
+
+				# Save any `alt` and `title` attributes because we may be interested in their contents
+				for node in dom.xpath("//*[@alt or @title]"):
+					for _, value in node.attrs.items():
+						xhtml = xhtml + f" {value} "
+
+				# Strip tags
+				xhtml = regex.sub(r"<[^>]+?>", " ", xhtml)
+
+				files_xhtml.append(xhtml)
+
+		except FileNotFoundError:
+			se.print_error(f"Couldn’t open file: [path][link=file://{filename}]{filename}[/][/].", plain_output=plain_output)
+			return_code = se.InvalidInputException.code
+
+		except se.SeException as ex:
+			se.print_error(str(ex) + f" File: [path][link=file://{filename}]{filename}[/][/].", plain_output=plain_output)
+			return_code = ex.code
+
+	# Create a list of unusual characters.
+	# We start with every character, and remove ones we know are valid in SE productions
+	unusual_character_set = "["
+	# Ignore basic ASCII u0000-u007e
+	unusual_character_set += "\u007f-\u009f"
+	# Ignore NO BREAK SPACE u00a0
+	unusual_character_set += "\u00a1"
+	# Ignore CENT SIGN and POUND SIGN u00a2-u00a3
+	unusual_character_set += "\u00a4-\u00af"
+	# Ignore DEGREE SYMBOL u00b0
+	unusual_character_set += "\u00b1-\u00bb"
+	# Ignore vulgar fractions u00bc-u00be
+	unusual_character_set += "\u00bf"
+	# Ignore standard accented letters u00c0-u00ff
+	unusual_character_set += "\u0100-\u0151"
+	# Ignore œ / Œ u0152-u0153
+	unusual_character_set += "\u0154-\u030c"
+	# Ignore COMBINING VERTICAL LINE ABOVE u030d
+	unusual_character_set += "\u030e-\u036f"
+	# Ignore basic Greek characters u0370-u03ff
+	unusual_character_set += "\u0400-\u1eff"
+	# Ignore extended Greek characters u1f00-u1fff
+	unusual_character_set += "\u2000-\u2009"
+	# Ignore HAIR SPACE u200a
+	unusual_character_set += "\u200b-\u2010"
+	# Ignore valid dashes u2011-u2014
+	unusual_character_set += "\u2015-\u2017"
+	# Ignore valid single quotes u2018-u2019
+	unusual_character_set += "\u201a-\u201b"
+	# Ignore valid double quotes u201c-u201d
+	unusual_character_set += "\u201e-\u2025"
+	# Ignore HORIZONTAL ELLIPSIS u2026
+	unusual_character_set += "\u2027-\u205f"
+	# Ignore WORD JOINER u2060
+	unusual_character_set += "\u2061-\u21a8"
+	# Ignore LEFTWARDS ARROW WITH HOOK u21a9 (used in endquotes)
+	unusual_character_set += "\u21aa-\u22ed"
+	# Ignore VERTICAL ELLIPSIS u22ee
+	unusual_character_set += "\u22ef-\u2e39"
+	# Ignore two-/three-em dashes u2e3a-u2e3b
+	unusual_character_set += "\u2e3c-\ufefe"
+	# Ignore ZERO WIDTH SPACE ufeff
+	unusual_character_set += "]"
+
+	unusual_character_regex = regex.compile(unusual_character_set)
```

Comment:
.compile() is no longer necessary with the regex library, it compile and caches regexes by default